entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/busybox/sh
```

Comment:
thought: the entry point script should not care for the exact path of the shell

It should not assume it is running in a busybox and expect a generic shell. Where Busybox keeps its shell is their business, but we should try a generic shebang. How about:
```sh
#!/usr/bin/env sh
```

This pattern usually locates executables without a flaw on POSIX-compatible systems. Could you try this one instead, please?